gamebook/game_input.py

```diff
@@ -0,0 +1,4 @@
+class Input(object):
+
+    def recieve_input(self):
```

Comment:
i think that it would be better to do in output, as there is already `option.show_user_inputs()` so we only need to print the string this method returns

Reply:
i'll add it

Reply:
on second thought, doing it in input is more efficient because it could be added to the `input()` prompt

Reply:
so it saves an entire line in `example.py`